tests/test_browser.py

```diff
@@ -1275,6 +1275,11 @@ def test_webgl_no_double_error(self):
   def test_preinitialized_webgl_context(self):
     self.btest('preinitialized_webgl_context.cpp', '5', args=['-s', 'GL_PREINITIALIZED_CONTEXT=1', '--shell-file', path_from_root('tests/preinitialized_webgl_context.html')])
 
+  # Primarily a compile-test for library_webgpu.
+  # TODO: is there a lighter-weight way to make compile tests?
```

Comment:
You can also put the file in `test_other` if you don't need to actually do a run in the browser. Or in `test_core` if you want to automatically run in different build modes.